pallets/rmrk-core/src/functions.rs

```diff
@@ -568,6 +583,7 @@ where
 		}
 	}
 
+	/// I don't think we use this at all? [delete this line if untrue, or this function if true]
```

Comment:
Looks like we don't use it. I think it can be removed.

Reply:
removing

Reply:
[196c9d8](https://github.com/rmrk-team/rmrk-substrate/pull/95/commits/196c9d8165e37bc0e2c93d132f456a0f049ea5f4)